axlearn/common/checkpointer.py

```diff
@@ -121,22 +122,47 @@ def filter_for_validation(structure):
         )
 
 
-# pylint: disable-next=redefined-builtin
-def save_tf_savables(value_map: Dict[str, Any], *, dir: str):
-    """Saves TF savables from `value_map` into `dir`."""
+def _upload_dir(src_dir_handle: tempfile.TemporaryDirectory, *, dst_dir: str):
+    """Upload a directory (non-recursively) from a temporary dir to dst_dir.
+
+    Temporary dir will be deleted after the upload is complete.
+    """
+    src_dir = src_dir_handle.name
+    if not tf.io.gfile.exists(dst_dir):
```

Comment:
nit -- `makedirs` should work even if it exists.

Reply:
Done.